### PR TITLE
Fix 1506 2

### PR DESCRIPTION
**Pull request by schakrad**
Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-rollouts/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this is a chore.
* [ ] The title of the PR is (a) [conventional](https://www.conventionalcommits.org/en/v1.0.0/), (b) states what changed, and (c) suffixes the related issues number. E.g. `"fix(controller): Updates such and such. Fixes #1234"`.  
* [x] I've signed my commits with [DCO](https://github.com/argoproj/argoproj)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My builds are green. Try syncing with master if they are not. 
* [ ] My organization is added to [USERS.md](https://github.com/argoproj/argo-rollouts/blob/master/USERS.md).

### CODE REVIEW

**Comment by harikrongali** [comment]
@schakrad is this PR ready for review?

**Comment by harikrongali** [comment]
@schakrad is this PR ready for review?

File: pkg/kubectl-argo-rollouts/info/analysisrun_info.go [hunk 1]
```diff
@@ -26,32 +26,48 @@ func getAnalysisRunInfo(ownerUID types.UID, allAnalysisRuns []*v1alpha1.Analysis
...
-			lastMeasurement := analysisutil.LastMeasurement(run, mr.Name)
-			if lastMeasurement != nil && lastMeasurement.Metadata != nil {
-				if jobName, ok := lastMeasurement.Metadata[job.JobNameKey]; ok {
-					jobInfo := rollout.JobInfo{
-						ObjectMeta: &v1.ObjectMeta{
-							Name: jobName,
-						},
-						Icon:   analysisIcon(lastMeasurement.Phase),
-						Status: string(lastMeasurement.Phase),
+			for _, meas := range analysisutil.ArrayMeasurement(run, mr.Name) {
```
Comment (huikang) [inline pkg/kubectl-argo-rollouts/info/analysisrun_info.go:43]: nitpick: spell out the `meas` for the sake of clarity

File: pkg/kubectl-argo-rollouts/info/analysisrun_info.go [hunk 2]
```diff
@@ -26,32 +26,48 @@ func getAnalysisRunInfo(ownerUID types.UID, allAnalysisRuns []*v1alpha1.Analysis
 				UID:               run.UID,
 			},
 		}
+		arInfo.SuccessCondition = run.Spec.Metrics[0].SuccessCondition
```
Comment (huikang) [inline pkg/kubectl-argo-rollouts/info/analysisrun_info.go:29]: Is it safe to assume `run.Spec.Metrics` is non-empty

File: pkg/kubectl-argo-rollouts/info/analysisrun_info.go [hunk 3]
```diff
@@ -26,32 +26,48 @@ func getAnalysisRunInfo(ownerUID types.UID, allAnalysisRuns []*v1alpha1.Analysis
...
+						Status:    string(meas.Phase),
+						StartedAt: meas.StartedAt,
 					}
-					arInfo.Jobs = append(arInfo.Jobs, &jobInfo)
+					arInfo.NonJobInfo = append(arInfo.NonJobInfo, &nonJobInfo)
 				}
 			}
 		}
 		arInfo.Icon = analysisIcon(run.Status.Phase)
 		arInfo.Revision = int32(parseRevision(run.ObjectMeta.Annotations))
```
Comment (huikang) [inline pkg/kubectl-argo-rollouts/info/analysisrun_info.go:70]: could you resolve the below warning?

**Pull request closed without merging**